lib/public/components/qcFlags/qcFlagsPagesButtons.js

```diff
@@ -57,15 +59,26 @@ export const qcFlagCreationPanelLink = (
         Success: (detectors) => {
             const userDetectors = detectors.find(({ id }) => id === dplDetectorId);
             if (userDetectors) {
-                return frontLink(
-                    content,
-                    pageName,
-                    { runNumber, dplDetectorId, ...options },
-                    {
-                        class: 'btn btn-primary',
-                        title: 'Create a QC flag',
-                    },
-                );
+                return remoteRun.match({
+                    NotAsked: () => null,
+                    Loading: () => null,
+                    Failure: () => tooltip(h('.f3', iconWarning()), 'An error has occurred. Unable to determine whether run is subject for QC'),
+                    Success: ({ runNumber, runQuality }) =>
+                        runQuality === RunQualities.BAD
+                            ? tooltip(
```

Comment:
Why using tooltip over title? I am not sure about the conventions here but I think we use title on disabled buttons

Reply:
applied

Reply:
No I meant using tooltip to display the message and not native title

Reply:
removed tooltip